Docs/troubleshooting.md

```diff
@@ -0,0 +1,236 @@
+
+
+#Troubleshooting
+Provide some tips and tricks to help customize and troubleshoot the solution.
```

Comment:
Change it to this: 
The aim of this document is to provide some tips and tricks to help customize and troubleshoot the solution.